notebooks/233-blip-visual-language-processing/233-blip-quantize-nncf.ipynb

```diff
@@ -0,0 +1,657 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    VISION_MODEL_OV_INT8 = Path("_int8.".join(VISION_MODEL_OV.as_posix().split(".")))
</pre><p>it can be simplified by</p><p>VISION_MODEL_OV_INT8 =  VISION_MODEL_OV.with_suffix("_int8.xml")  or </p><p>VISION_MODEL_OV_INT8 = Path(VISION_MODEL_OV.name.replace('.xml', '_int8.xml"))</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>